diabetes_regression/training/train_aml.py

```diff
@@ -120,10 +123,10 @@ def main():
         train_args = {}
 
     # Log the training parameters
```

Comment:
Can you add a logging statement that looks like:

`logging.info("training_parameters", train_args)` ?

This will help us see all the parameters for this training session inside the custom dimensions in the `traces` table.

Reply:
To update this convo, the correct way would be like this:
```
properties = {'custom_dimensions': train_args}
 
# Use properties in logging statements
logger.info('action', extra=properties)```

Reply:
Thanks Ivan, implemented this already. 👍